test/integration/commands/makedminCommandTest.ts

```diff
@@ -0,0 +1,78 @@
+﻿import { strict as assert } from "assert";
+
+import config from "../../../src/config";
+import { newTestUser } from "../clientHelper";
+import { PowerLevelAction } from "matrix-bot-sdk/lib/models/PowerLevelAction";
+import { LogService } from "matrix-bot-sdk";
+import { getFirstReaction } from "./commandUtils";
+
+describe("Test: The make admin command", function () {
+    afterEach(function () {
+        this.moderator?.stop();
+        this.userA?.stop();
+        this.userB?.stop();
+    });
+
+    it('Mjölnir make the bot self room administrator', async function () {
+        this.timeout(60000);
+        const mjolnir = config.RUNTIME.client!;
+        const mjolnirUserId = await mjolnir.getUserId();
+        const moderator = await newTestUser({ name: { contains: "moderator" } });
+        this.moderator = moderator;
+
+        await moderator.joinRoom(config.managementRoom);
+        LogService.debug("makeadminTest", `Joining managementRoom: ${config.managementRoom}`);
+        let targetRoom = await moderator.createRoom({ invite: [mjolnirUserId] });
```

Comment:
Could you also invite another user? We'll test that the user is *not* made admin.

Reply:
Done.